<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">* {
bakcground-color: white;
font-family: poppins;
}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>poppins</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Check In</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>19</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>poppins</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Check Out</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbCheckIn">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>poppins</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>AM/PM</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>AM</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>PM</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="cmbCheckOut">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>20</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>poppins</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>AM/PM</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>AM</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>PM</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="btnOK">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>181</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>poppins</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
	background-color: &quot;green&quot;;
	color: white;
	border-radius: 5px
</string>
   </property>
   <property name="text">
    <string>Filter</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClear">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>250</y>
     <width>161</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>poppins</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
	background-color: &quot;#344273&quot;;
	color: white;
	border-radius: 5px
</string>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnMissing">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>poppins</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
	background-color: &quot;darkred&quot;;
	color: white;
	border-radius: 5px
</string>
   </property>
   <property name="text">
    <string>Missing</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
